homeassistant/components/insteon/insteon_entity.py

```diff
@@ -75,7 +76,14 @@ def name(self):
     @property
     def extra_state_attributes(self):
         """Provide attributes for display on device card."""
-        return {"insteon_address": self.address, "insteon_group": self.group}
+        ramprate = self.get_device_property(RAMP_RATE)
+        onamount = self.get_device_property(ON_LEVEL)
+        return {
+            "insteon_address": self.address,
+            "insteon_group": self.group,
+            "insteon_ramprate": ramprate,
+            "insteon_onlevel": onamount,
```

Comment:
Do we need those attributes here? Why not expose them as separate sensors?

Reply:
I think they make sense as config entities but wanted to first fix a bug (turn-on) as a getting started pull request. Having them shown is pretty useful for diagnosing switch issues and ensures I've done things right.

Reply:
> I think they make sense as config entities but wanted to first fix a bug (turn-on) as a getting started pull request.

Well, if that is the idea, it would also mean that merging this PR means a breaking change up ahead in the next PR. Instead, we should not add them to avoid the breaking change.

Reply:
I don't quite understand this either. The bug fix here would remain as is and adding additional entities further on wouldn't affect this afaik. The Insteon on_level configuration is the level at which the physical button turns on when the light is off. Changing that value programmatically is a nice-to have that wouldn't impact this code. Changing ramp-rate imho is far more important because the Insteon switches sporadically mess up their ramp values but again isn't a breaking change to this code is it?

Reply:
I admit I'm new to HomeAssistant and could be totally wrong but I'm just trying to fix a bug for now...

Reply:
I get that you are trying to fix a bug, but I don't see what adding attributes has to do with bug fixing?

Reply:
We should not add the attributes per @frenck comment. I am working on a change to introduce these properties as entities per the new configuration entities model.

Reply:
ok, the attribute addition was just for debugging and it felt in context with the insteon_group info. I'll take it out. It was mainly to ensure the code was actually doing what I thought physically. I'll remove it.